### PR TITLE
fixes related to integrated services

### PR DESCRIPTION
**Pull request by lpuskas**
| Q               | A
| --------------- | ---
| Bug fix?        |yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Bugfixes related to integrated services:
- cluster related flags taken into account while operating integrated services
- fixed activation request assembly

### Why?
- cluster flags were ignored by the tool when managing integrated services
- the request read from files (or stdin) were lost while activating integrated services

### CODE REVIEW

**Comment by lpuskas** [comment]
the update flow also seems to be broken (the fix is underway)

File: internal/cli/command/cluster/integratedservice/services/activate.go [hunk 1]
```diff
@@ -80,22 +73,25 @@ func runActivate(
...
 
-		if err := showActivateEditor(m, request); err != nil {
+		if err = showActivateEditor(m, request); err != nil {
 			return errors.WrapIf(err, "failed during showing editor")
 		}
-
 	} else {
-		if err := readActivateReqFromFileOrStdin(options.filePath, request); err != nil {
+		var reqFromFile pipeline.ActivateClusterFeatureRequest
+		if err = readActivateReqFromFileOrStdin(options.filePath, &reqFromFile); err != nil {
```
Comment (siliconbrain) [inline internal/cli/command/cluster/integratedservice/services/activate.go]:
```go
		request = new(pipeline.ActivateClusterFeatureRequest)
		if err := readActivateReqFromFileOrStdin(options.filePath, request); err != nil {
```

Comment (lpuskas) [reply internal/cli/command/cluster/integratedservice/services/activate.go]: thx fixed

**Comment by lpuskas** [comment]
> the update flow also seems to be broken (the fix is underway)

the fix has been pushed

**Pull request merged**